Dockerfile

```diff
@@ -1,11 +1,6 @@
-FROM golang:1.18 as workspace
-WORKDIR /go/src/github.com/testcontainers/moby-ryuk
-COPY go.mod go.sum ./
-RUN go mod download
-COPY . ./
-RUN make build
-
-FROM alpine:3.13.6
+FROM alpine:3.16.1
 RUN apk --no-cache add ca-certificates
```

Comment:
Do we require this line anymore?

Reply:
I'd say the ca-certificates were present in the old base image, probably to allow SSL connections, so not sure if we should remove the package now

Reply:
Hm, I do not understand which SSL connections that could be. TC connects to the plain socket and the Docker build stage does not fetch any data.

Reply:
TBH I'm not sure why it was there before these changes. If you agree, let's keep it here for this PR and create a follow-up discussion on the real need for the ca-certificates package.

Reply:
OC, sure.